wom/src/main/java/wdl4s/parser/MemoryUnit.java

```diff
@@ -2,14 +2,10 @@
 
 public enum MemoryUnit {
     Bytes(1, "B"),
-    KiB(1L << 10, "KiB", "Ki"),
-    MiB(1L << 20, "MiB", "Mi"),
-    GiB(1L << 30, "GiB", "Gi"),
-    TiB(1L << 40, "TiB", "Ti"),
-    KB(1000, "KB", "K"),
-    MB(KB.bytes * 1000, "MB", "M"),
-    GB(MB.bytes * 1000, "GB", "G"),
-    TB(GB.bytes * 1000, "TB", "T");
+    KB(1L << 10, "KB", "K", "KiB", "Ki"),
+    MB(1L << 20, "MB", "M", "MiB", "Mi"),
+    GB(1L << 30, "GB", "G", "GiB", "Gi"),
+    TB(1L << 40, "TB", "T", "TiB", "Ti");
```

Comment:
🙌 (Though I'm sure someone somewhere is going to disagree with this over-simplicity)

Reply:
Hmm, this looks like a bit shift, but why is that in units of 10? I'm sure it makes sense but 😕

Reply:
Hmm to be honest I just kept what was already there for the `xiB` values. There are tests that verify that 1KB now is 1024 bytes so it seems right ?